addons/modules/CfgVehicles.hpp

```diff
@@ -0,0 +1,461 @@
+class CfgVehicles {
+    class Logic;
+    class Module_F: Logic {
+        class ArgumentsBaseUnits;
+        class ModuleDescription;
+    };
+
+    class CBA_ModuleAttack: Module_F {
```

Comment:
You can add `author = "WiredTiger";` to each of the three module classes and it'll show up in the editor when one browses modules.